.github/workflows/style.yaml

```diff
@@ -71,6 +71,9 @@ jobs:
     - name: Check Policies
       run: scripts/ci/jobs/policy-checks.sh
 
+    - name: Check proto
+      run: scripts/ci/jobs/check-gogoproto.sh
```

Comment:
@mtodor @rhybrillou  Do you think we need something more?

Reply:
@janisz I'm not able to find this script and tagged change. Is this removed or moved to followup PR?

Reply:
I've removed it as it's not needed if we remove migrate protos step in qa
- https://github.com/stackrox/stackrox/pull/10257